src/zenml/integrations/github/code_repositories/github_code_repository.py

```diff
@@ -202,3 +215,51 @@ def check_remote_url(self, url: str) -> bool:
             return True
 
         return False
+
+
+def create_symlink_in_local_repo_copy(
+    symlink_dst: str, symlink_src: str
+) -> None:
+    """This function attempts to create a symbolic link at `symlink_dst` that points to `symlink_src`.
+
+    If a file or directory already exists at `symlink_dst`, it will
+    be removed before the symbolic link is created.
+
+    Args:
+        symlink_dst: The path where the symbolic link should be created.
+        symlink_src: The path that the symbolic link should point to.
+
+    Raises:
```

Comment:
To pass our CI, we'll need to remove those as they're actually not raised anymore, but a warning is logged instead.